src/time.c

```diff
@@ -28,6 +28,48 @@ extern "C"
 #include "rcutils/error_handling.h"
 #include "rcutils/snprintf.h"
 
+double
+rcutils_nanoseconds_to_seconds(const int64_t nanoseconds)
+{
+  // scale the nanoseconds separately for improved accuracy
+  int64_t sec, nsec;
+  nsec = (nanoseconds % 1000000000l);
```

Comment:
just a minor suggestion. can we have like `RCUTILS_NANOSECS_PER_SEC` for `1000000000` and use it else where? e.g `RCUTILS_S_TO_NS`, `RCUTILS_NS_TO_S`. the same thing can go to miliseconds and microseconds.

Reply:
Yeah seems like a great metric for users to have, especially if they design their own time-based algorithms and want standardization. I would figure it'll get used enough to be worth having 3 definitions.